### PR TITLE
feat(use-context-menu-controller): update focusOnContextMenu

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [ ] Updated Storybook documents for component
- [x] Wrote test codes for composable, utils and etc.
- [ ] Tested with console(if usages are changed)

### Description

image

### CODE REVIEW

File: src/hooks/context-menu-controller/__tests__/index.test.ts [hunk 1]
```diff
@@ -59,48 +72,64 @@ describe('Context Menu Controller', () => {
...
+                contextMenuRef: ref<typeof PContextMenu|null>(null),
+                visibleMenu: ref(true),
+                menu: ref(menuItems),
+            }));
+            const { focusOnContextMenu } = result as UseContextMenuControllerReturns;
+            it('focusOnContextMenu() should focus on context menu element.', async () => {
+                expect(document.activeElement?.id).toBeFalsy();
+                focusOnContextMenu();
+                await Vue.nextTick();
+                expect(document.activeElement?.id).toBeTruthy();
```
Comment (yuda110) [inline src/hooks/context-menu-controller/__tests__/index.test.ts:131]: activeElement checks focused element!

File: src/hooks/context-menu-controller/index.ts [hunk 2]
```diff
@@ -77,7 +72,6 @@ export const useContextMenuController = ({
         showContextMenu,
         hideContextMenu,
         focusOnContextMenu,
-        focusOutFromContextMenu,
```
Comment (yuda110) [inline src/hooks/context-menu-controller/index.ts:80]: question: why is this deleted?

**Review by yuda110 — APPROVED** [review]
LGTM!

File: src/hooks/context-menu-controller/index.ts (hunk 2 above)
Comment (WANZARGEN) [reply src/hooks/context-menu-controller/index.ts:80]: I think this is not necessary.. :)
